src/gl.rs

```diff
@@ -293,229 +298,252 @@ pub trait Gl {
+        fn create_shader(&self, shader_type: GLenum) -> GLuint;
+        fn delete_shader(&self, shader: GLuint);
+        fn detach_shader(&self, program: GLuint, shader: GLuint);
+        fn link_program(&self, program: GLuint);
+        fn clear_color(&self, r: f32, g: f32, b: f32, a: f32);
+        fn clear(&self, buffer_mask: GLbitfield);
+        fn clear_depth(&self, depth: f64);
+        fn clear_stencil(&self, s: GLint);
+        fn flush(&self);
+        fn finish(&self);
+        fn get_error(&self) -> GLenum;
+        fn stencil_mask(&self, mask: GLuint);
+        fn stencil_mask_separate(&self, face: GLenum, mask: GLuint);
+        fn stencil_func(&self, func: GLenum, ref_: GLint, mask: GLuint);
+        fn stencil_func_separate(&self, face: GLenum, func: GLenum, ref_: GLint, mask: GLuint);
+        fn stencil_op(&self, sfail: GLenum, dpfail: GLenum, dppass: GLenum);
+        fn stencil_op_separate(&self, face: GLenum, sfail: GLenum, dpfail: GLenum, dppass: GLenum);
+        fn egl_image_target_texture2d_oes(&self, target: GLenum, image: GLeglImageOES);
+        fn generate_mipmap(&self, target: GLenum);
+        fn insert_event_marker_ext(&self, message: &str);
+        fn push_group_marker_ext(&self, message: &str);
+        fn pop_group_marker_ext(&self);
+        fn fence_sync(&self, condition: GLenum, flags: GLbitfield) -> GLsync;
+        fn client_wait_sync(&self, sync: GLsync, flags: GLbitfield, timeout: GLuint64);
+        fn wait_sync(&self, sync: GLsync, flags: GLbitfield, timeout: GLuint64);
+        fn delete_sync(&self, sync: GLsync);
+        fn texture_range_apple(&self, target: GLenum, data: &[u8]);
+        fn gen_fences_apple(&self, n: GLsizei) -> Vec<GLuint>;
+        fn delete_fences_apple(&self, fences: &[GLuint]);
+        fn set_fence_apple(&self, fence: GLuint);
+        fn finish_fence_apple(&self, fence: GLuint);
+        fn test_fence_apple(&self, fence: GLuint);
+
+        // GL_ARB_blend_func_extended
+        fn bind_frag_data_location_indexed(
+            &self,
+            program: GLuint,
+            color_number: GLuint,
+            index: GLuint,
+            name: &str,
+        );
+        fn get_frag_data_index(
+            &self,
+            program: GLuint,
+            name: &str,
+        ) -> GLint;
 
-    // GL_ARB_blend_func_extended
-    fn bind_frag_data_location_indexed(
-        &self,
-        program: GLuint,
-        color_number: GLuint,
-        index: GLuint,
-        name: &str,
-    );
-    fn get_frag_data_index(
-        &self,
-        program: GLuint,
-        name: &str,
-    ) -> GLint;
+        fn alias_point_size_range(&self) -> (GLfloat, GLfloat);
+        fn alias_line_width_range(&self) -> (GLfloat, GLfloat);
 
-    fn alias_point_size_range(&self) -> (GLfloat, GLfloat);
-    fn alias_line_width_range(&self) -> (GLfloat, GLfloat);
+        // Returns the the maximum supported width and height of the viewport.
```

Comment:
I also disagree with this patch if it leads to docblocks being removed, though I kill this method anyway in my own patch.